content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -718,9 +732,12 @@ is turned on.
   more than one default is specified, the admission plugin forbids the creation of
   all PVCs.
 * If the admission plugin is turned off, there is no notion of a default
-  StorageClass. All PVCs that have no `storageClassName` can be bound only to PVs that
-  have no class. In this case, the PVCs that have no `storageClassName` are treated the
-  same way as PVCs that have their `storageClassName` set to `""`.
+  StorageClass. All PVCs that have `storageClassName` set to `""` can be
+  bound only to PVs that have `storageClassName` also set to `""`.
+  However, PVCs with missing `storageClassName` can be updated later once
+  default StorageClass becomes available. If the PVC gets updated it will no
+  longer bind to PVs that have `storageClassName` also set to `""`.
+  See `RetroactiveDefaultStorageClass` [feature](/docs/concepts/storage/persistent-volumes/#retroactive-storageclass-assignment) for more details.
```

Comment:
```suggestion

See [Retroactive StorageClass assignment](#retroactive-storageclass-assignment) for more details.
```

Reply:
When writing this I've checked which form should be used and I found both where the feature/component name is **not** part of the link:

https://github.com/kubernetes/website/blob/13a3118ca5d206b95c295d3f2dcd4aa1a11778bb/content/en/docs/concepts/storage/persistent-volumes.md?plain=1#L58

And where it **is** part of the link:

https://github.com/kubernetes/website/blob/13a3118ca5d206b95c295d3f2dcd4aa1a11778bb/content/en/docs/concepts/storage/persistent-volumes.md?plain=1#L709

So I don't know which one is better. More opinions are welcome :)

Reply:
I recommend writing it my way (I'm a tech lead for SIG Docs). In this case, the name of the feature gate is an unimportant detail.